apps/dashr-yield-curve/app.R

```diff
@@ -0,0 +1,428 @@
+library(plotly)
+library(dplyr)
+library(tidyr)
+library(data.table)
+library(purrr)
+library(quantmod)
+library(magrittr)
+library(dash)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(glue)
+library(jsonlite)
+library(rlang)
+
+#setwd("~/Dash apps/dashr-yield-curve")
+
+appName <- Sys.getenv("DASH_APP_NAME")
+pathPrefix <- sprintf("/%s/", appName)
+
+
+Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+           DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+setwd(sprintf("/app/apps/%s", appName))
+
+l. <- list
+
+#Helper functions
+
+genMarksEmpty <- function(n){
+  l <- list(glue(''))
+  names(l) <- 'label'
+  m <- list(l)
+  names(m) <- glue('{n}')
+  return(m)
+}
```

Comment:
How about in one line `setNames(list(setNames(list(glue('')),'label')),  glue('{n}'))`?